notify/pushover/pushover.go

```diff
@@ -147,6 +147,7 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	parameters.Add("expire", fmt.Sprintf("%d", int64(time.Duration(n.conf.Expire).Seconds())))
 	parameters.Add("device", tmpl(n.conf.Device))
 	parameters.Add("sound", tmpl(n.conf.Sound))
+	parameters.Add("ttl", fmt.Sprintf("%d", int64(time.Duration(n.conf.TTL).Seconds())))
```

Comment:
The parameter should be added only if the ttl is > 0.

Reply:
Hi @simonpasquier could it be something like: 
```
	newttl := int64(time.Duration(n.conf.TTL).Seconds())
	if newttl > 0 {
		parameters.Add("ttl", fmt.Sprintf("%d", newttl))
	}
```

Reply:
yes

Reply:
ok thanks for the feedback, push the change, when you have the time pls review